### PR TITLE
PHEE-387 Create fineract stub APIs in AMS_mifos for successful response.

### PR DESCRIPTION
**Pull request by logoutdhaval**
## Description

[PHEE-387 Create fineract stub APIs in AMS_mifos for successful response.](https://fynarfin.atlassian.net/browse/PHEE-387?atlOrigin=eyJpIjoiZmQyMDcxMDgxOTFkNGI3YTlkOTBkMzc3ODU0OWVjY2UiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by logoutdhaval** [comment]
@fynmanoj LGTM

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -30,3 +30,22 @@ ams:
...
+spring:
+  profiles:
+    active: "bb,fin12"
+  jackson:
+    date-format: org.mifos.connector.fineractstub.RFC3339DateFormat
+    serialization:
+      WRITE_DATES_AS_TIMESTAMPS: false
+
+
+springfox:
```
Comment (fynmanoj) [inline src/main/resources/application.yml:43]: We are using spring docs for other services, can you tell me why spring fox needs to be adopted here?

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -30,3 +30,22 @@ ams:
...
+  documentation:
+    open-api:
+      v3:
+        path: /api-docs
+server:
+  servlet:
+    contextPath: /fineract
+  port: 7070
```
Comment (fynmanoj) [inline src/main/resources/application.yml:51]: Is this port exposed from container?

File: src/main/resources/application-fin12.yml [hunk 3]
```diff
@@ -40,6 +40,19 @@ ams:
       host: http://localhost:8443
       repayment-path: /fineract-provider/api/v1/interoperation/transactions/{accountNumber}/loanrepayment
 
-
+mock-service:
+  run-on-mock: true
```
Comment (fynmanoj) [inline src/main/resources/application-fin12.yml:44]: What happened to ams.local.enabled flag? Aren't we supposed to use that here?

File: src/main/resources/application-fin12.yml [hunk 4]
```diff
@@ -1,12 +1,12 @@
 camel:
-  server-port: 5000
+  server-port: 5008
```
Comment (fynmanoj) [inline src/main/resources/application-fin12.yml:2]: Why does this port has to be changed?

File: src/main/java/org/mifos/connector/fineractstub/model/SavingsDeposit.java [hunk 5]
```diff
@@ -0,0 +1,179 @@
+package org.mifos.connector.fineractstub.model;
+
```
Comment (fynmanoj) [inline src/main/java/org/mifos/connector/fineractstub/model/SavingsDeposit.java:2]: Can you mark generated files with an annotation and exclude them from check style checks

**Review by fynmanoj — CHANGES_REQUESTED** [review]

**Review by fynmanoj — CHANGES_REQUESTED** [review]
Requesting changes

File: src/main/java/org/mifos/connector/fineractstub/model/SavingsDeposit.java (hunk 5 above)
Comment (logoutdhaval) [reply src/main/java/org/mifos/connector/fineractstub/model/SavingsDeposit.java:2]: done

File: src/main/resources/application.yml (hunk 2 above)
Comment (logoutdhaval) [reply src/main/resources/application.yml:51]: yes

File: src/main/resources/application.yml (hunk 1 above)
Comment (logoutdhaval) [reply src/main/resources/application.yml:43]: Tried with springdocs but getting import issues.

**Pull request closed without merging**